consensus/consensus-types/src/block.rs

```diff
@@ -109,7 +109,10 @@ impl Block {
             None => 0,
             Some(payload) => match payload {
                 Payload::InQuorumStore(pos) => pos.proofs.len(),
-                Payload::DirectMempool(txns) => txns.len(),
+                Payload::DirectMempool(_txns) => 0,
+                // TODO: this is used only for counters, hence the inaccurate number for now should not matter for now;
+                //       but we should fix this
```

Comment:
given the above, return len here is correct, remove comment.

Reply:
Ok